prow/git/git.go

```diff
@@ -488,22 +488,3 @@ func (i *Repo) ShowRef(commitlike string) (string, error) {
 	}
 	return strings.TrimSpace(string(out)), nil
 }
-
-// FetchFromRemote runs a git fetch command for the specified remote.
-// Pass the destination as org/repo format since the actual URI will be constructed
-// using the existing credentials and will always use the https protocol.
-func (r *Repo) FetchFromRemote(orgRepo, branch string) error {
```

Comment:
Why remove it?

Reply:
Do we actually want this in the v1 client? I can write the adapter also. I did this in previous versions of this PR.

Reply:
I would assume that if we have code, we have it for a reason and we shouldn't just randomly remove it

Reply:
Yea I added this in a previous PR, but it seems its better to have the whole thing in v2 only. Shall we keep it? I can make changes.